vtex/toolbelt
# Use new VTEX ID login flow

#### What is the purpose of this pull request?
Use new VTEXID login flow.

#### How should this be manually tested?
Install the following version:
```
yarn global add vtex@2.105.1-beta.6
```
Check if `vtex login` is working

#### Types of changes

#### Chores checklist
